egs/wsj/s5/steps/nnet3/decode.sh

```diff
@@ -80,7 +80,7 @@ thread_string=
 [ $num_threads -gt 1 ] && thread_string="-parallel --num-threads=$num_threads"
 
 mkdir -p $dir/log
-[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj || exit 1;
+[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh --per-utt $data $nj || exit 1;
```

Comment:
OK thanks.

Vijay

On Jul 21, 2016 8:21 PM, "Daniel Povey" <notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet3/decode.sh
> <https://github.com/kaldi-asr/kaldi/pull/875#discussion_r71810668>:
>
> > @@ -80,7 +80,7 @@ thread_string=
> >  [ $num_threads -gt 1 ] && thread_string="-parallel --num-threads=$num_threads"
> >
> >  mkdir -p $dir/log
> > -[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj || exit 1;
> > +[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh --per-utt $data $nj || exit 1;
>
> We've been waiting too long for the per-utt splitting to be working. I'll
> change the data-splitting script right now and check it in within the hour.
> After that you can set something like sdata=$dir/split${nj}utt instead of
> sdata=$dir/split${nj} and it will work. Dan On Thu, Jul 21, 2016 at 5:19
> PM, Vijayaditya Peddinti < notifications@github.com> wrote:
> … <#m_-6281962373127726343_>
> In egs/wsj/s5/steps/nnet3/decode.sh <#875 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/875#discussion_r71810489>>: > @@
> -80,7 +80,7 @@ thread_string= > [ $num_threads -gt 1 ] &&
> thread_string="-parallel --num-threads=$num_threads" > > mkdir -p $dir/log
> > -[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj
> || exit 1; > +[[ -d $sdata && $data/feats.scp -ot $sdata ]] ||
> split_data.sh --per-utt $data $nj || exit 1; I intended it to be permanent
> as the ASpIRE decodes take several hours when done per speaker. I will
> remove it from the script as soon as I get access to a computer. Vijay …
> <#m_-858257127955897913_> On Jul 21, 2016 8:08 PM, "Daniel Povey"
> ***@***.***> wrote: In egs/wsj/s5/steps/nnet3/decode.sh <#875
> <https://github.com/kaldi-asr/kaldi/pull/875> (comment) <#875 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/875#discussion_r71809522>>>: >
> @@ -80,7 +80,7 @@ thread_string= > [ $num_threads -gt 1 ] &&
> thread_string="-parallel --num-threads=$num_threads" > > mkdir -p $dir/log
> > -[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj
> || exit 1; > +[[ -d $sdata && $data/feats.scp -ot $sdata ]] ||
> split_data.sh --per-utt $data $nj || exit 1; Was the addition of this
> --per-utt flag intended to be temporary? Until issue #778
> <https://github.com/kaldi-asr/kaldi/issues/778> <#778
> <https://github.com/kaldi-asr/kaldi/issues/778>> <#778
> <https://github.com/kaldi-asr/kaldi/issues/778> <#778
> <https://github.com/kaldi-asr/kaldi/issues/778>>> is addressed, this is
> not safe to commit as it would interfere with jobs that do require
> splitting per-speaker. — You are receiving this because you authored the
> thread. Reply to this email directly, view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/875/files/2a81f157021f8b7adf2d175b945cc40b4e794e6e#r71809522>,
> or mute the thread <
> https://github.com/notifications/unsubscribe-auth/ADtwoLg2PCygEZLJlaVHee-SM5h61RNmks5qYAntgaJpZM4JCR8D>
> . — You are receiving this because you were mentioned. Reply to this email
> directly, view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/875/files/2a81f157021f8b7adf2d175b945cc40b4e794e6e#r71810489>,
> or mute the thread <
> https://github.com/notifications/unsubscribe-auth/ADJVu_eKrztA8tWJ7xCcwHOFlmxoA8G7ks5qYAyYgaJpZM4JCR8D>
> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/875/files/2a81f157021f8b7adf2d175b945cc40b4e794e6e#r71810668>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADtwoN_YYDfMEGv6B0fPPrZcE4jwBCsEks5qYA0kgaJpZM4JCR8D>
> .
>